src/MessageMap.ts

```diff
@@ -6,12 +6,14 @@ type Direction = "d2t" | "t2d";
 /** Handles mapping between message IDs in discord and telegram, for message editing purposes */
 export class MessageMap {
 	private _map: Map<Bridge, any>;
-	private _messageTimeout: number;
+	private _messageTimeoutAmount: number;
+	private _messageTimeoutUnit: moment.unitOfTime.DurationConstructor;
 
-	constructor(messageTimeout: number) {
+	constructor(messageTimeoutAmount: number, messageTimeoutUnit: string) {
```

Comment:
Can't the unit argument be of DurationConstructor here? Then you don't need the "as" below